helm/cas-metabase/templates/cron-prod-test-restore.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.restoreFromProd.enable }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "cas-metabase.fullname" . }}-prod-test-restore
+  labels:
+{{ include "cas-metabase.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-upgrade,pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "-5"
+spec:
+  suspend: true # This cron job is intended to be triggered manually
+  schedule: "* * * * *"
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+  activeDeadlineSeconds: 3600
+  template:
+    metadata:
+      labels: {{ include "cas-metabase.labels" . | nindent 8 }}
+        release: cas-metabase
+    spec:
+      activeDeadlineSeconds: 3600
+      restartPolicy: Never
+      containers:
+        - env:
+            - name: MB_USER
+              value: metabase
+            - name: MB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: database-password
+                  name: cas-metabase
+            - name: MB_DATABASE
+              value: metabase
+            - name: MB_HOST
+              value: cas-metabase-patroni
+            - name: MB_PORT
+              value: "5432"
+            - name: SUPERUSER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password-superuser
+                  name: cas-metabase-patroni
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: "/home/postgres/credentials/credentials.json"
+            - name: GOOGLE_APPLICATION_CREDENTIALS_JSON
+              valueFrom:
+                secretKeyRef:
+                  key: credentials.json
+                  name: {{ .Values.existingGcpSecret }}
+            - name: WALG_GS_PREFIX
+              value: "gs://9212c9-prod-mb-backups/wal"
```

Comment:
`9212c9` could be replaced by the release's namespace prefix to avoid hardcoding